doc/dev/developer_guide/basic-workflow.rst

```diff
@@ -243,15 +243,16 @@ Fixing the bug in the local working copy
 #. **Fixing the bug itself**
 
    This guide cannot tell you how to fix the bug that you have chosen to fix.
-   This guide assumes that you know what required improvement, and that you
-   know what to do to provide that improvement.
+   This guide assumes that you have identified an area that required
+   improvement, and that you know how to make that improvement.
 
-   It might be that your fix is simple and requires only minimal testing. But
-   that's unlikely. It is more likely that the process of fixing your bug will
-   be iterative and will involve trial, error, skill, and patience. 
+   It might be that your fix is simple and that it requires only minimal
+   testing. But that's unlikely. It is more likely that the process of fixing
```

Comment:
unlikely (unless only documentation is updated).

Reply:
Fair enough.